package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua

```diff
@@ -35,6 +35,7 @@ function M.is_outdoor_device()
 		'tplink,cpe510-v3',
 		'tplink,eap225-outdoor-v1',
 		'tplink,wbs210-v2',
+		'ubnt,unifiac-lite',
```

Comment:
The AC Lite is an indoor device.

Reply:
resolved in d3a2c657f912b1d7354c962fc91bcc5ba437f739